plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java

```diff
@@ -126,7 +126,9 @@
     // import the projects
     progress.setWorkRemaining(10 * projectFiles.size() + 10);
     List<IProject> projects = new ArrayList<>(projectFiles.size());
+    System.out.printf("Importing %d projects:\n", projectFiles.size());
```

Comment:
What's this going to do if it ships? Show up in the console I guess?

Reply:
This is a test utility, so it won't ship?